CHANGELOG.md

```diff
@@ -21,7 +29,7 @@ Pylance's copy of Pyright has been updated from 1.1.291 to 1.1.292.
     -   Bug Fix: Fixed a bug that resulted in incorrect type evaluation when accessing instance methods directly from a `None` instance
     -   Bug Fix: Fixed a regression that resulted in a crash when resolving overloads with different numbers of parameters and with with arguments that evaluate to `Any`
 
-## 2023.2.10 (1 February 2023) Release
+## 2023.1.10 (1 February 2023) Release
```

Comment:
Oh looks like bill fixed this here, but I didn't in the pyrx bits. I'll put this back